src/cmd-build

```diff
@@ -74,11 +75,12 @@ if [ $# -eq 0 ]; then
     set -- qemu
 fi
 
-build_qemu=
+build_derived=
 for target in "$@"; do
     case $target in
         ostree) ;;
-        qemu) build_qemu=1;;
+        qemu) build_derived=qemu;;
```

Comment:
We can't do both in one shot here, but that's fine. What are your thoughts on splitting building the ostree into a separate command later?